keras_nlp/models/albert/albert_tokenizer.py

```diff
@@ -61,6 +61,13 @@ class AlbertTokenizer(SentencePieceTokenizer):
 
     # Detokenization.
     tokenizer.detokenize(tf.constant([[2, 14, 2231, 886, 2385, 3]]))
+
```

Comment:
~10 lines up from here, you should be instantiating the tokenizer with `from_preset`

Reply:
the line is ```tokenizer = keras_nlp.models.AlbertTokenizer(proto="model.spm")```